extensions/datasource/deployment/src/main/java/io/quarkus/datasource/deployment/devservices/DevUIDatasourceProcessor.java

```diff
@@ -31,4 +35,26 @@ public DevConsoleTemplateInfoBuildItem devConsoleInfo(
     DevConsoleRouteBuildItem devConsoleCleanDatabaseHandler(DatabaseRecorder recorder) {
         return new DevConsoleRouteBuildItem("reset", "POST", recorder.devConsoleResetDatabaseHandler());
     }
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    CardPageBuildItem create(DataSourcesBuildTimeConfig dataSourceBuildTimeConfig) {
+        CardPageBuildItem card = new CardPageBuildItem("Datasources");
+
+        List<String> names = new ArrayList<>();
+        names.add("<default>");
+        names.addAll(dataSourceBuildTimeConfig.namedDataSources.keySet());
+        Collections.sort(names);
+        card.addBuildTimeData("datasources", names);
+
+        card.addPage(Page.webComponentPageBuilder().title("Reset")
+                .componentLink("datasources-component.js")
```

Comment:
name should follow conversion qwc-extensionName-pageName so `qwc-datasources-reset`

Reply:
Any reason? I never had an issue with a custom file name.

Reply:
Yes it should work with any name as long as it's unique. Using the extension name somewhat guarantee uniqueness, but the real reason is actually for something I want to do in the future when I have some time. I want to "automatically" add the jsonRPC const to the web component (somehow - I have a few ideas). Knowing that I can just use the extension name makes that possible. So as an extension developer there will be less boilerplate.